_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Social Media Use in 2021"
+description: |
+  % of adults using each social media.
+categories: "2022" 
+author: Álvaro Sanz 
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(fig.showtext=TRUE)
+
+
+library(tidyverse)
+library(ggplot2)
+library(lemon)
+library(showtext)
```

Comment:
Keep these into a separate chunk without the `include=FALSE` option, because we want to see them in the post.